docs/source/en/model_doc/seamless_m4t.md

```diff
@@ -15,7 +15,8 @@ specific language governing permissions and limitations under the License.
 ## Overview
 
 The SeamlessM4T model was proposed in [SeamlessM4T — Massively Multilingual & Multimodal Machine Translation](https://dl.fbaipublicfiles.com/seamless/seamless_m4t_paper.pdf) by the Seamless Communication team from Meta AI.
-This is the version 1 release of the model. For the updated version 2 release, refer to the [Seamless M4T v2 docs](./seamless_m4t_v2.md).
+
+This is the version 1 release of the model. For the updated version 2 release, refer to the [Seamless M4T v2 docs](https://huggingface.co/docs/transformers/main/model_doc/seamless_m4t_v2).
```

Comment:
It might be a good idea to format it in bold to make it stand out more, WDYT ?

Reply:
Formatted the "version X" statements in bold https://github.com/huggingface/transformers/pull/27905/commits/415ff23108748df4262618278ad4bd44b48bf9ab